src/main/java/seedu/address/model/recipe/Ingredient.java

```diff
@@ -28,10 +30,11 @@
     public Ingredient(String name, String quantity) {
         requireNonNull(name);
         requireNonNull(quantity);
-        checkArgument(isValidIngredient(name), MESSAGE_CONSTRAINTS);
-        checkArgument(isValidIngredient(quantity), MESSAGE_CONSTRAINTS);
+        // checkArgument(isValidIngredient(name), MESSAGE_CONSTRAINTS);
```

Comment:
we have to incorporate these in our code

Reply:
if I'm not wrong they help to verify that the arguments are non-null

Reply:
> if I'm not wrong they help to verify that the arguments are non-null

I think the ones he deleted is to verify the name is valid, but since this version has no constraints on name/ingredient at this point, it should be fine.